src/components/App/App.css

```diff
@@ -46,9 +45,32 @@
   background-color: white;
 }
 
-.prescription-page {
-  display: flex;
-  justify-content: center;
-  height: 80%;
-  width: 45%;
-}
```

Comment:
I removed this and instead styled the prescription size, vs having it fill the parent. This just took away a layer of complexity.